Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml

```diff
@@ -0,0 +1,4175 @@
+  playbookInputQuery:
+- key: FurtherInvestigation
+  value:
+    simple: "False"
+  required: true
+  description: |-
+    Determines whether an incident should be further investigated if similar previous false positive incidents were found.
+    Possible values:True/False. Default: False.
+  playbookInputQuery:
+- key: AutoBlockIndicators
+  value:
+    simple: "True"
+  required: true
+  description: |-
+    Determine whether the given indicators be automatically blocked, or should the user be given the option to choose.
+    Possible values:True/False. Default: False.
+    If set to False - no prompt will appear, and all provided indicators will be blocked automatically.
+    If set to True - the user will be prompted to select which indicators to block.
+  playbookInputQuery:
+- key: BlockIndicators_UserVerification
+  value:
+    simple: "False"
+  required: false
+  description: |-
+    Determine whether the blocking of any indicator requires the verification of the user.
+    Possible values:True/False. Default: False.
+  playbookInputQuery:
+- key: EarlyContainment
+  value:
+    simple: "True"
+  required: true
+  description: |-
+    Whether early containment should be allowed when the IP address is known to be malicious.
+    Possible values:True/False. Default: True.
+  playbookInputQuery:
+- key: AutoIsolateEndpoint
+  value:
+    simple: "False"
+  required: true
+  description: |-
+    Whether to isolate the initiating endpoint automatically if the investigation verdict is malicious.
+    Possible values:True/False. Default: False.
+  playbookInputQuery:
+- key: Alert_ID
+  value:
+    complex:
+      root: PaloAltoNetworksXDR.Incident.alerts
+      filters:
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: PaloAltoNetworksXDR.Incident.alerts.name
+            iscontext: true
+          right:
+            value:
+              simple: Large Upload
+          ignorecase: true
+      accessor: alert_id
+      transformers:
+      - operator: uniq
+  required: false
+  description: The Cortex XDR alert ID.
+  playbookInputQuery:
+- key: Transferred_Data _Threshold
+  value:
+    simple: "150"
+  required: true
+  description: |-
+    Specify the uploaded data threshold volume (in MB) from which large upload alerts should be investigated.
+    By setting a threshold, you will be able to determine which large upload alerts require investigation.
+    Default value: 150 (MB).
+  playbookInputQuery:
+- key: FWApps_Processes_Whitlist
+  value:
+    simple: ip,tcp,udp,ssl,syslog,quic,Chrome.exe,Firefox.exe,Opera.exe,Safari.exe,iexplore.exe,msedge.exe,brave.exe
+  required: false
+  description: A list of known and authorized FW application IDs and processes used in the organization.
+  playbookInputQuery:
+outputs: []
+tests:
+- Test XDR Playbook general commands
+- Test Playbook - Cortex XDR - Endpoint Investigation
+- Test XDR Playbook
+fromversion: 6.8.0
```

Comment:
we should make it higher as we support new content only from 6.10

Reply:
changed